app/src/main/java/com/CodeNaroNa/vendor/relief/Activity/SignUp.java

```diff
@@ -109,6 +112,7 @@ public void onComplete(@NonNull Task<AuthResult> task) {
                             int selectedId = userselected.getCheckedRadioButtonId();
                             selection = (RadioButton) findViewById(selectedId);
                             Toast.makeText(getApplicationContext(),selection.getText().toString(),Toast.LENGTH_SHORT).show();
```

Comment:
comment this

Reply:
Should I comment all three lines i.e. 112,113,114 or just Toast line(114)?

Reply:
no we are using selection later in the code i guess,